src/imresh/io/writeOutFuncs/writeOutFuncs.cpp

```diff
@@ -26,6 +26,7 @@
 #include <algorithm>
 #ifdef IMRESH_DEBUG
 #   include <iostream>              // std::cout, std::endl
+
```

Comment:
Meh happens. I think I needed iomanip or similar and then deleted my temporary debug code again, but not cleanly